themes/color-theme.json

```
@@ -309,18 +309,18 @@
     "sideBarTitle.foreground": "_fg-inactive-symbol",
     "statusBar.background": "_base",
     "statusBar.border": "_transparent",
-    "statusBar.debuggingBackground": "_pine",
-    "statusBar.debuggingBorder": "_transparent",
-    "statusBar.debuggingForeground": "#fff",
     "statusBar.foreground": "_fg-subtle",
+    "statusBar.debuggingBackground": "_debug-bg",
+    "statusBar.debuggingBorder": "_transparent",
+    "statusBar.debuggingForeground": "_debug-fg",
```

use alphabetical order

```suggestion
    "statusBar.debuggingBackground": "_debug-bg",
    "statusBar.debuggingBorder": "_transparent",
    "statusBar.debuggingForeground": "_debug-fg",
    "statusBar.foreground": "_fg-subtle",
```